### PR TITLE
[MIG][12.0] product_variant_inactive

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

File: product_variant_inactive/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2015 Abdessamad HILALI <abdessamad.hilali@akretion.com>
```
Comment (sebastienbeau) [inline product_variant_inactive/models/product_template.py]: wrong copyright

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

**Comment by sebastienbeau** [comment]
only small copyright issue, else the migration is ok for me

File: product_variant_inactive/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product Variant Inactive",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "website": "http://www.akretion.com",
```
Comment (emagdalenaC2i) [inline product_variant_inactive/__manifest__.py]:
```suggestion
    'website': 'https://github.com/OCA/product-attribute',
```

**Review by emagdalenaC2i — COMMENTED** [review]
LGTM Just a small change in the manifest

File: product_variant_inactive/__manifest__.py (hunk 2 above)
Comment (kevinkhao) [reply product_variant_inactive/__manifest__.py]: Fixed, thanks

**Review by emagdalenaC2i — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]

**Pull request merged**